packages/@webex/internal-plugin-metrics/test/unit/spec/call-diagnostic/call-diagnostic-metrics.ts

```diff
@@ -5,7 +5,7 @@ import {WebexHttpError} from '@webex/webex-core';
 import CallDiagnosticMetrics from '../../../../src/call-diagnostic/call-diagnostic-metrics';
 import CallDiagnosticLatencies from '../../../../src/call-diagnostic/call-diagnostic-metrics-latencies';
 import * as Utils from '../../../../src/call-diagnostic/call-diagnostic-metrics.util';
```

Comment:
See the positive comment in the other test file. This approach (absolute/relative imports) breaks existing scripts in both `beta` and `next`, such as `yarn distsrc` and similar (review `./CONTRIBUTING.md`) on `beta`, and all runtime tests on `next`. Please consider switching these to use aliased imports if you could

(note that this may be out of scope for this pull request).

Reply:
I've refactored this.

Reply:
Seems a bit silly that I have to export those classes from the plugin in order to test them. Not sure if there's a way around that.. but I don't mind too much.